### ISSUE TITLE
TBD: Can database.retrieveSPRCacheItem use the "spr" table rather the "geojson" table

### ISSUE THREAD

**Issue by thisisaaronland** [whosonfirst/go-whosonfirst-spatial-sqlite#2]
Blocked on:

* https://github.com/whosonfirst/go-whosonfirst-sqlite-features/issues/12
* https://github.com/whosonfirst/go-whosonfirst-sqlite-features-index/issues/4

Also, not clear that it's worth it since the `go-whosonfirst-spatial/cache.CacheItem` interface requires geometries:

* https://github.com/whosonfirst/go-whosonfirst-spatial/blob/master/cache/cache.go

But since the geometries are only necessary if we are converting an SPR to GeoJSON maybe loading the geometries can be delayed?

**Comment by thisisaaronland** [comment]
```
Time to retrieve 85633041-alt-naturalearth-display-terrestrial-zoom6 from database, 10.656911ms
Time to load feature for 85633041-alt-naturalearth-display-terrestrial-zoom6, 597.103034ms
Time to create spr cache item for 85633041-alt-naturalearth-display-terrestrial-zoom6, 935.211463ms
Time to retrieve 85633041-alt-naturalearth-display-terrestrial-zoom6, 1.447165883s
Time to check 85633041:naturalearth-display-terrestrial-zoom6, 1.505028073s
```

**Comment by thisisaaronland** [comment]
Fixed as of v0.0.8
